tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -638,6 +705,38 @@ def test_create_pgm_input_asym_loads(mock_init_array: MagicMock, two_pp_objs, co
     assert converter.pgm_input_data["asym_load"] == mock_init_array.return_value
 
 
+def test_create_pgm_input_sym_loads__asym() -> None:
```

Comment:
Only delta loads are tested here. Are wye loads tested somewhere else? If so, I think this function should have a more meaningful name, e.g. `test_create_pgm_input_sym_loads__asym_delta` or something like that. If not, shouldn't that be implemented?